packages/material-ui/src/TablePagination/TablePagination.test.js

```diff
@@ -374,4 +374,29 @@ describe('<TablePagination />', () => {
       );
     });
   });
+
+  describe('prop: SelectProps', () => {
+    it('does allow manual label ids', () => {
```

Comment:
Just curious: what about this test asserts that the underlying component respects the provided IDs?

Reply:
The `toHaveAccessibleName`. The `Typography` component that received the mismatched `id` is not part of the public API here. The `Select` itself always respected `id` and `labelId` from `SelectProps`. But it didn't match.